app/assets/javascripts/analytics/_pageViews.js

```diff
@@ -1,7 +1,33 @@
 (function (GOVUK) {
   GOVUK.GDM.analytics.pageViews = {
     'init': function () {
+      this.setCustomDimensions()
       GOVUK.analytics.trackPageview();
-    }
+    },
+
+    'setCustomDimensions': function() {
+      var customDimensionData,
+          dimensionValToBeSet,
+          h1Val;
+      
+      if (this.createOpportunityResponseRegex.test(GOVUK.GDM.analytics.location.pathname())) {
+        customDimensionData = $('[data-analytics=customDimension]').first().data('custom-dimension');
+        h1Val = $('h1').first().text();
+
+        // If we find a custom dimension data attribute then we will send that as our custom dimension
+        if (customDimensionData) {
+          dimensionValToBeSet = customDimensionData;
+        // Else check if the page title text indicates the supplier is indeed able to apply
+        } else if (h1Val.indexOf("You can’t {{ inhibited_action }} this opportunity") == -1 && h1Val.indexOf("Apply for ") >= 0) {
```

Comment:
It doesn't seem like this condition is needed. From my understanding of the different page states possible, those without the custom dimension can be assumed to be those where a supplier is eligible.

If that's the case, we also don't need to cache the dimension in `dimensionValToBeSet` as we can return the function as soon as a dimension is set.